RecoJets/JetProducers/plugins/VirtualJetProducer.cc

```diff
@@ -752,11 +752,10 @@ void VirtualJetProducer::writeJets(edm::Event& iEvent, edm::EventSetup const& iS
         // Here it is assumed that fjJets_ is in decreasing order of pT,
         // which should happen in FastjetJetProducer::runAlgorithm()
         jetArea = M_PI;
-        //rij is properly initialized above
-        [[clang::suppress]]
         RIJ* distance = rij[ijet];
         for (unsigned jJet = 0; jJet < ijet; ++jJet) {
-          distance[jJet].first = std::sqrt(reco::deltaR2(etaJ[ijet], phiJ[ijet], etaJ[jJet], phiJ[jJet])) * orParam_;
+          [[clang::suppress]] distance[jJet].first =
+              std::sqrt(reco::deltaR2(etaJ[ijet], phiJ[ijet], etaJ[jJet], phiJ[jJet])) * orParam_;
```

Comment:
looks like false positive warning. `etaJ[ijet]` is initialized properly in the above block